db/database.go

```diff
@@ -2479,6 +2488,15 @@ func (db *DatabaseContext) StartOnlineProcesses(ctx context.Context) (returnedEr
 	}
 	db.backgroundTasks = append(db.backgroundTasks, bgtSyncTime)
 
+	db.AttachmentMigrationManager = NewAttachmentMigrationManager(db)
+	// if we have collections requiring migration, run the job
+	if len(db.RequireAttachmentMigration) > 0 && !db.BucketSpec.IsWalrusBucket() {
+		err := db.AttachmentMigrationManager.Start(ctx, nil)
```

Comment:
I think this might need to be `Run` because we don't want to take the database online while attachments are in a mixed format? Or are we allowing that?